api/src/main/java/org/pmiops/workbench/cohorts/CohortMaterializationService.java

```diff
@@ -107,19 +142,13 @@ private ColumnConfig findPrimaryKey(TableConfig tableConfig) {
     throw new IllegalStateException("Table lacks primary key!");
   }
 
-  private TableQueryAndConfig getTableQueryAndConfig(FieldSet fieldSet,
+  private TableQueryAndConfig getTableQueryAndConfig(@Nullable TableQuery tableQuery,
```

Comment:
I don't follow the relationship between this PR and the TableQuery becoming nullable.

Reply:
Before we passed in fieldSet, which was nullable but we hadn't annotated it that way. Now we pass in TableQuery, which is also nullable.